spec/stored_queries/taxonomy/taxonomy_search_species.yaml

```diff
@@ -1,3 +1,5 @@
+# DEPRECATED. See taxonomy_search_species_strains and taxonomy_search_species_strains_no_sort
```

Comment:
do you know if anyone / anything is actually using this?

Reply:
I really don't think so